uniffi_bindgen/src/bindings/python/templates/RustBufferHelper.py

```diff
@@ -1,17 +1,13 @@
-# Helpers for lifting/lowering primitive data types from/to a bytebuffer.
+# Helpers for reading/writing types from/to a bytebuffer.
```

Comment:
On a side note, I tried our calling these "read/write" rather than "lift_from/lower_into", and TBH I think it makes things a bit simpler to understand.

Reply:
I'd like to do this with all the backends.

Reply:
If we like this, I'd be happy to go through the other backends and make them all consistent. @linacambridge @tarikeshaq @eoger feel free to thumbs-up or thumbs-down this comment for feedback :-)

Reply:
That sounds really good. It did take me a while to wrap my head around the difference between `lift`, `liftFrom`, `lower`, and `lowerInto` at first, and especially since the thing that does the lifting and lowering is called a "reader" and a "writer", changing the names to read and write seems natural.

Reply:
Aye, I think we can call it a bit of an experiment in terminology that didn't work out (which, it's good to notice these things and change them before they get too entrenched!)